### PR TITLE
fix(docs): Update troubleshooting and include docker commands

### PR DESCRIPTION
**Pull request by ojeytonwilliams**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

As discussed in #35841 the troubleshooting section needed updating.

### CODE REVIEW

File: docs/how-to-setup-freecodecamp-locally.md [hunk 1]
```diff
@@ -474,23 +474,29 @@ If you are stuck, and need help, let us know by asking in the ['Contributors' ca
 There might be an error in the console of your browser or in Bash / Terminal / Command Line that will help identify the problem. Provide this error message in your problem description so others can more easily identify the issue and help you find a resolution.
 
 ### Troubleshooting
-If the app launches but you are encountering errors with the UI itself, for example if fonts are not being loaded or if the code editor is not displaying properly, you may try the following troubleshooting steps at least once:
+If the app launches but you are encountering errors with the UI itself, for example if fonts are not being loaded or if the code editor is not displaying properly, try the following if you are using docker:
```
Comment (SammyHass) [inline docs/how-to-setup-freecodecamp-locally.md]: Just changing some of the phrasing as well as separating the different methods more clearly. This makes it slightly easier to follow along.

```suggestion
Upon launching the app, if you are encountering issues with the UI such as the fonts not loading or the code editor not displaying, try the following depending on your situation.

#### Using Docker
```

File: docs/how-to-setup-freecodecamp-locally.md [hunk 2]
```diff
@@ -474,23 +474,29 @@ If you are stuck, and need help, let us know by asking in the ['Contributors' ca
...
 
 # Seed the database
-npm run seed
+npm run docker:seed
 
 # Restart the application
+npm run docker:develop
+```
+
+If you are not using docker, the process is very similar, but uses different versions of each command:
```
Comment (SammyHass) [inline docs/how-to-setup-freecodecamp-locally.md]:
```suggestion
### Without Docker
```

**Review by sammyhass — CHANGES_REQUESTED** [review]
Looks great! Please just make the following adjustments to the docs. Just improves on language and readability.

Thanks!

File: docs/how-to-setup-freecodecamp-locally.md (hunk 1 above)
Comment (RandellDawson) [reply docs/how-to-setup-freecodecamp-locally.md]: This suggestion is not proper grammar and definitely does not feel like it makes it easier to read in my opinion.  It is fine starting with "If" and not fine to start with a prepositional phrase like "Upon launching the app".  How about something more like:
> If the app launches but you are encountering UI errors such as fonts not being loaded or the code editor not displaying properly, see the following depending on your local setup:

### Using Docker:

File: docs/how-to-setup-freecodecamp-locally.md (hunk 2 above)
Comment (RandellDawson) [reply docs/how-to-setup-freecodecamp-locally.md]: Continuing the flow from above:
```suggestion
Not Using Docker
```

**Comment by ojeytonwilliams** [comment]
Thanks @SammyHass and @RandellDawson for the suggestions.  I went with Randell's wording for the opening paragraph as I felt it fit better with the overall tone of the document.  Sammy's point about verbosity was a good one, but I used `**Stuff**` rather than a header to be consistent with earlier in the document.

**Comment by RandellDawson** [comment]
@SammyHass The OP has made some changes.  Do you think these are better than the original version?

**Review by sammyhass — APPROVED** [review]
Looks good! Nice one. Thanks, @RandellDawson, your wording was undoubtedly better than mine, you are clearly very astute 👍

**Review by sammyhass — APPROVED** [review]

**Review by RandellDawson — APPROVED** [review]
I think these changes look good now.  Let's get another mod to review before merging.

**Review by Manish-Giri — COMMENTED** [review]
Not sure if the [scripts](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/docs/how-to-setup-freecodecamp-locally.md#step-2-install-dependencies) have changed, but is `npm run docker:init` no longer required?

**Comment by ojeytonwilliams** [comment]
@Manish-Giri as far as I understand it, it needs to be run, but only once.  It sets the ownership of various directories to allow a less privileged user (called `node`) to install packages in them.

**Review by Manish-Giri — APPROVED** [review]
@ojeytonwilliams Thank You for your contribution. 💯

**Pull request merged**